app/src/main/java/ar/com/wolox/android/features/homepage/viewpager/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package ar.com.wolox.android.features.homepage.viewpager
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+
+class ViewPagerAdapter(supportFragmentManager: FragmentManager) : FragmentPagerAdapter(supportFragmentManager) {
+
+    private val mFragmentList = ArrayList<Fragment>()
```

Comment:
I don't really like using hungarian notation here. Also, if you are declaring an ArrayList, I think it's better to not use the list word in the variable name. I would leave it just like `private val fragments`